x-pack/plugins/ml/public/data_frame/pages/data_frame_new_pivot/directive.tsx

```diff
@@ -24,7 +24,7 @@ type CreateSearchItems = () => { indexPattern: StaticIndexPattern };
 import { KibanaContext } from '../../common';
 import { Page } from './page';
 
-module.directive('mlNewDataFrame', ($injector: InjectorService) => {
+module.directive('mlNewDataFrame', ($injector: InjectorService, kbnBaseUrl: string) => {
```

Comment:
mixture of injector styles. `kbnBaseUrl` could be got from `$injector`

Reply:
Fixed in c638d28.